include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -7099,6 +7099,38 @@
       "capabilities" : [ "AtomicFloat32AddEXT", "AtomicFloat64AddEXT" ],
       "extensions" : [ "SPV_EXT_shader_atomic_float_add" ],
       "version" : "None"
+    },
+    {
+      "opname" : "OpAtomicFMinEXT",
+      "class"  : "Atomic",
+      "opcode" : 5614,
```

Comment:
Please move this up so the instructions appear in numeric order.

Reply:
Done.